asyncapi-client.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib,spec/factories}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
-  s.add_dependency "rails", "~> 4.0"
+  s.add_dependency "rails", "= 5.1.4"
```

Comment:
Let's loosen our version for `rails`. We don't want this gem to be incompatible after a minor or patch update.